testsuite/tests/properties/character/test.py

```diff
@@ -37,5 +37,8 @@ def text_identity(s=T.String):
 
```

Comment:
Please move this commit before 458e6b23454297f4db798b254adc445ef956d184, otherwise the test will fail on that commit.